pyproject.toml

```diff
@@ -1,7 +1,92 @@
 [build-system]
-requires = ["setuptools", "wheel"]
+requires = ["setuptools>=67.0.0"]
 build-backend = "setuptools.build_meta"
 
+
+[project]
+name = "pipenv"
+authors = [
+  {name = "Pipenv maintainer team", email = "distutils-sig@python.org"},
```

Comment:
https://github.com/pypa/pipenv/pull/5900#pullrequestreview-1606631997

Reply:
I think this should be a new issue. At some point, issue drift starts to occur and pull requests never get merged. The focus of this issue was to merely move content from `setup.py` and `setup.cfg` to `pyproject.toml`. That required a few additional changes where the files were referenced and due to the change in associated build processes. I don't believe, however, we should hold up the PR or require another run of CI/CD for this at this time, given how long the PR has been open and how many times it has been rebased. Additionally, I don't believe there is a known value to substitute at this time.

Reply:
#5906

Reply:
I wasn't calling out we had to fix it to merge this work, just I noticed it in reviewing it.